build/src/lib.rs

```diff
@@ -227,21 +367,25 @@ impl BinaryBuilder {
 
         self.outputs.push(tokenizer_out);
         self.outputs.push(rules_out);
+        Ok(())
     }
 
     /// Creates a new binary builder. `language_codes` must be in ISO 639-1 (two-letter) format.
-    /// If `language_codes` is `None`, uses all supported languages.
+    /// If `language_codes` is `&[]`, uses all supported languages.
```

Comment:
This is good! I thought about using an empty slice to signify all languages but went for an `Option` instead. On second thought I like the empty slice better.